Formula/sdl2_image.rb

```diff
@@ -41,11 +42,11 @@ def install
 
     system "./autogen.sh" if build.head?
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
+    system "./configure", *std_configure_args,
                           "--disable-imageio",
                           "--disable-jpg-shared",
                           "--disable-png-shared",
+                          "--disable-stb-image",
```

Comment:
stb-image was made the default in 2.6.0 rather than libjpeg/libpng (https://github.com/libsdl-org/SDL_image/releases/tag/release-2.6.0)

So, it is also possible to drop `jpeg-turbo`/`libpng` dependencies and not set `--disable-stb-image`.

Not sure how much performance difference is.